### PR TITLE
docs: add baseurl build arg

### PR DESCRIPTION
**Pull request by dvdksn**
Signed-off-by: David Karlsson <35727626+dvdksn@users.noreply.github.com>

### CODE REVIEW

**Comment by dvdksn** [comment]
cc @milosgajdos @Jamstah

**Comment by Jamstah** [comment]
Yes, I think combine things.

Also, we should be testing this by pushing to main in a personal org or something, and when that works, we can push it to main here.

**Comment by milosgajdos** [comment]
@dvdksn do you mind closing this one now that @Jamstah added these changes to #4102

File: dockerfiles/docs.Dockerfile [hunk 1]
```diff
@@ -16,8 +16,9 @@ COPY --from=hugo $GOPATH/bin/hugo /bin/hugo
 WORKDIR /src
 
 FROM build-base AS build
+ARG DOCS_BASEURL
 RUN --mount=type=bind,rw,source=docs,target=. \
-    hugo --gc --minify --destination /out
+    hugo --gc --minify --destination /out -b $DOCS_BASEURL
```
Comment (thaJeztah) [inline dockerfiles/docs.Dockerfile:21]: what happens when building without passing a base-URL? will `-b` without value cause it to fail?

(what's the default if the option is not set? or should the ARG have a default (eg `localhost`) ?

Comment (milosgajdos) [reply dockerfiles/docs.Dockerfile:21]: Can we please discuss this in https://github.com/distribution/distribution/pull/4102. I'd like @dvdksn to close this PR as the changes have been move to ☝️

**Pull request closed without merging**